kiwi/bootloader/config/grub2.py

```diff
@@ -44,7 +44,7 @@ class BootLoaderConfigGrub2(BootLoaderConfigBase):
     def post_init(self, custom_args):
         self.custom_args = custom_args
         arch = platform.machine()
-        if arch == 'x86_64':
+        if arch == 'x86_64' or arch.startswith('ppc64'):
```

Comment:
could we make this more explicit ? like

```python
elif arch == 'ppc64' or arch == 'ppc64le'
```

or are there so many different ppc archs that we better match the way you did it ?